Translations/zh-Hans.lproj/Localizable.strings

```diff
@@ -125,7 +125,7 @@
 "unknown" = "未知";
 "Unknown error" = "未知错误";
 "Error code %d using \"%@\"\n%@" = "错误码 %d 使用“%@”\n%@";
-"Keka was unable to create the extraction package" = "Keka was unable to create the extraction package";
+"Keka was unable to create the extraction package" = "Keka 无法创建压缩包";
```

Comment:
This is about the "extraction" package, where the extraction will go. Is that one right?

Reply:
Is there has a detailed context description?

Reply:
While extracting a compressed file a ".kextraction" package is created, and there the extraction goes while unfinished. Once finished, it's moved to the proper extracted files. Imagine "the extraction package" as a temporary place to store the extraction files while they are being extracted.

![Screenshot 2021-01-25 at 10 54 35](https://user-images.githubusercontent.com/480163/105690096-dfd4ce80-5efb-11eb-982d-f048909de64e.png)

Reply:
OK, I am understanding it. Let me think about how to translate it of "extraction package".